Systems/bushkit.xml

```diff
@@ -132,6 +132,94 @@ Extra weight and drag due to bush wheels, floats and aircraft with 180 hp engine
         </fcs_function>
     </channel>
 
+    <channel name="ski-reduced-friction">
+
+        <switch name="static-friction-front-ski">
+            <default value=".08"/>
+            <test logic="AND" value=".02">
+                bushkit EQ 5
+                /fdm/jsbsim/ground/solid EQ 1
```

Comment:
Can this be changed to `ground/solid EQ 1`? (question, not a suggestion)

Reply:
@onox yes, I think so, nice catch.